wrapanapi/lenovo.py

```diff
@@ -137,6 +137,9 @@ def get_server_mac_address(self, server_name, timeout=600):
 
         return server['macAddress']
 
+    def get_server_power_status(self, requester=None):
```

Comment:
I'd much prefer a `server_name` argument instead of `requestor`, or just `name` for more consistency with existing API.

Note the abstract methods in base take a `vm_name` or `name` parameter in all cases where one is necessary.

`requestor`, especially without any docblock, is very ambiguous to me here. Is it a pytest Request object, a AttrDict with name key, or an object of some other type?